handlers/userbot_creation.py

```
@@ -0,0 +1,207 @@
+from aiogram import Router, F
+from aiogram.types import CallbackQuery, Message
```

Ruff (0.12.2)

`aiogram.types.Message` imported but unused

Remove unused import: `aiogram.types.Message`

(F401)